providers/google/tests/unit/google/cloud/hooks/test_gcs.py

```diff
@@ -39,7 +39,7 @@
 from airflow.providers.google.cloud.hooks import gcs
 from airflow.providers.google.cloud.hooks.gcs import _fallback_object_url_to_object_name_and_bucket_name
 from airflow.providers.google.common.consts import CLIENT_INFO
-from airflow.utils import timezone
+from airflow.sdk import timezone
```

Comment:
Here I change utils to sdk because airflow.utils.timezone has been deprecated. Currently, mypy can't recognize it. 
<img width="846" height="208" alt="截圖 2025-09-26 上午10 13 53" src="https://github.com/user-attachments/assets/12356fd0-8417-44b0-b27c-ab7baa9e81cc" />